cassiopeia/simulator/BirthDeathFitnessSimulator.py

```diff
@@ -147,6 +149,85 @@ def __init__(
         self.collapse_unifurcations = collapse_unifurcations
         self.random_seed = random_seed
 
+        # useful for resuming a simulation, perhaps under different pressures.
+        self.initial_tree = initial_tree
+
+    def initialize_tree(self, names) -> nx.DiGraph:
+        """initializes a tree (nx.DiGraph() object with one node). Auxiliary data for each node is grabbed from self (initial conditions / params) or hardcoded
+
+        Args: names is a generator (function object that stores internal state) that will be used to generate names for the tree nodes
+        Output: tree (DiGraph object with one node, the root)
+                root (name of root node in tree)
+        """
+        if self.initial_tree:
+            tree = self.initial_tree.get_tree_topology()
+            for node in self.initial_tree.nodes:
+                tree.nodes[node]['birth_scale'] = self.initial_tree.get_attribute(node, 'birth_scale')
```

Comment:
What if initial tree doesn't have a 'birth_scale' attribute? Maybe set to initial_birth_scale?